src/services/did-service.service.ts

```diff
@@ -20,7 +20,7 @@ export const register = async (
 ): Promise<DidDocument> => {
   const hcsMessages = new HcsMessageCollectorService();
   const didKeypair = await DidKeypairModel.findById(did);
-  const privateKey = PrivateKey.fromString(didKeypair.privateKey);
+  const privateKey = PrivateKey.fromString(didKeypair!.privateKey!);
```

Comment:
@vijayshiyani how about adding a conditional clause
```
if (!privateKey) { throw new Error('...') }
```
? I think that would be a nicer way of handling this specific case. I have noticed that there are more cases like this in other services too.

Reply:
fix